fsharp-backend/src/LibBackend/UserDB.fs

```diff
@@ -114,6 +114,8 @@ and typeCheck (db : RT.DB.T) (obj : RT.DvalMap) : RT.DvalMap =
         | RT.TDate, RT.DDate _ -> value
         // CLEANUP use the inner type
         | RT.TList _, RT.DList _ -> value
+        // CLEANUP use the inner type
+        | RT.TTuple _, RT.DTuple _ -> value
```

Comment:
I don't think this comment actually makes sense, since tuples should have heterogeneous types